client/src/views/home/HomePageHeader.vue

```diff
@@ -59,7 +59,7 @@
         </ion-button>
         <ion-button
           fill="solid"
-          v-show="!isElectron()"
+          v-show="isElectron()"
```

Comment:
Nop, we don't want to show this button on electron while the client area is not ready.

Reply:
fixed